specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -3144,6 +3197,30 @@
           "description": "region of the Gallery Image Version."
         }
       }
+    },
```

Comment:
Hold your horses. 
How is this new property different from the property just above it called LatestGalleryImageVersion? https://github.com/Azure/azure-rest-api-specs/pull/23633/files#diff-2d57b1e3ef500806be7f276354fc7ae66b634ce4a9ba619f0b983855859acf35R3186 

The two descriptions for these 2 properties are literally identical. Why id your property being added at all? How is it different?

Reply:
it's nested. LatestGalleryImageVersion is a property of latestVersion.

latestVersion is an ARM resource which has id, name, type. LatestGalleryImageVersion is the property of it, which contains only region and version name.

Reply:
I think I should change the description of them, to make them clear